### PR TITLE
Taimoor/added examples

### PR DESCRIPTION
**Pull request by Taimoormk**
Issue:

## What I did

I simply removed the example of `reactPakistan` and added four mores at the bottom of the list, as `reactPakistanCommons`, `reactPakistanUI`, `reactPakistanIcons` & `reactPakistanLogos`.

## How to test

Simply route to the examples page and you will see the newly added content there.

- Is this testable with Jest or Chromatic screenshots? - no
- Does this need a new example in the kitchen sink apps? - no
- Does this need an update to the documentation? - no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: docs/src/pages/examples/_examples.yml [hunk 1]
```diff
@@ -204,3 +198,27 @@ pxblue:
...
+  title: React Pakistan - React Icon Collection
+  description: React Icon collection library is a categorised sets of Reactified versions of SVG icon components, designed to be friendly for developers, you can customise icons on the fly, e.g its colors, size etc.
+  demo: https://taimoormk.github.io/react-icon-collection/?path=/docs/react-pakistan-intro--page
+  site: https://www.npmjs.com/package/@react-pakistan/react-icon-collection
+reactPakistanLogo:
+  thumbnail: react-logo-collection.jpg
+  title: React Pakistan - React Logo Collection
+  description: React Logo collection library is a categorised sets of Reactified versions of SVG logo components, designed to be friendly for developers, you can customise icons on the fly, e.g its colors, size etc.
+  demo: https://taimoormk.github.io/react-logo-collection/?path=/story/react-pakistan-intro--page
+  site: https://www.npmjs.com/package/@react-pakistan/react-logo-collection
```
[the remark this answers is not in the document]
Comment (Taimoormk) [reply docs/src/pages/examples/_examples.yml:224]: @ndelangen Most certainly it makes sense and it's a great suggestion/feature as well. I am keen on to try it out and list that one composed library instead.

Also, perhaps you could point me in the right direction as well, I have 4 storybook libraries for mobile development too built using React Native, currently I am unable to host that library on a web based mobile emulator, do you happen to know any solution for that matter?

[the remark this answers is not in the document]
Comment (ndelangen) [reply docs/src/pages/examples/_examples.yml:224]: I don't really have a good suggestion for react-native right now, sorry. Let me know if you find something?

**Comment by Taimoormk** [comment]
Hi @ndelangen Yes certainly, I have setup an open source repo for this matter, https://github.com/react-pakistan/react-storybook-composed

Followed the instruction from the doc you shared above, I am able to reach to this point now.
Screenshot 2020-06-11 at 08 48 22

Already published stories aren't loading up as shown, care to have a look at this file please, https://github.com/react-pakistan/react-storybook-composed/blob/dev/.storybook/main.js

File: docs/src/pages/examples/_examples.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Taimoormk) [reply docs/src/pages/examples/_examples.yml:224]: No worries, and yes sure I am passively looking for that solution, will update you and this thread, once I happen to find one.

**Comment by ndelangen** [comment]
`url: 'https://taimoormk.github.io/react-logo-collection/?path=/story/logos--bitcoincolorlogos',` isn't a valid ref url, it should be `https://taimoormk.github.io/react-logo-collection`.

For larger storybooks like this you'll want to create a `stories.json` that's hosted alongside the deployed storybook.

you can use `yarn sb extract` to generate this file.

You'll want to do this for every referenced storybook. This should make the loading of refs nearly instant.

**Comment by Taimoormk** [comment]
> `url: 'https://taimoormk.github.io/react-logo-collection/?path=/story/logos--bitcoincolorlogos',` isn't a valid ref url, it should be `https://taimoormk.github.io/react-logo-collection`.
> 
> For larger storybooks like this you'll want to create a `stories.json` that's hosted alongside the deployed storybook.
> 
> you can use `yarn sb extract` to generate this file.
> 
> You'll want to do this for every referenced storybook. This should make the loading of refs nearly instant.

I was trying to run `npx sb extract` but seems like `extract command is not supported by the cli yet, as shown in the screenshot below;

Screenshot 2020-06-18 at 14 55 19

**Comment by ndelangen** [comment]
@Taimoormk our script checks if puppeteer is already installed/working on your system, if not tries to install it.

It seems installing fails on your machine because of permission issues.

**Comment by ndelangen** [comment]
We removed the visual examples from the website if you'd like you can add examples here:
https://github.com/storybookjs/storybook/blob/next/docs/get-started/examples.md

**Pull request closed without merging**